docs/cli-usage.md

```diff
@@ -79,6 +79,11 @@ Options:
                                   rules tailored to this project; your project
                                   URL will be used to log in to the Semgrep
                                   registry.
+
+                                  To run multiple rule files simultaneously, use -f
+                                  before every YAML, URL, or Semgrep registry
+                                  entry name. For example:
+                                  `semgrep -f your/java.net.url. -f path/secrets`.
```

Comment:
```suggestion
                                  `semgrep --config p/python --config myrules/myrule.yaml`.
```
Prefer to use long flags in docs, IMO. also I think it's useful to provide a registry example

Reply:
Thanks! Good point.

Reply:
Ok, actually, now I am confused - so we can run more rules by using any command again? @ievans

Reply:
@ievans Updated in CLI too: https://github.com/returntocorp/semgrep/pull/4751

Reply:
@ievans @minusworld updated here in the docs as well.